bitnami/rabbitmq/values.yaml

```diff
@@ -472,7 +485,7 @@ podAffinityPreset: ""
 ## @param podAntiAffinityPreset Pod anti-affinity preset. Ignored if `affinity` is set. Allowed values: `soft` or `hard`
 ## Ref: https://kubernetes.io/docs/concepts/scheduling-eviction/assign-pod-node/#inter-pod-affinity-and-anti-affinity
 ##
-podAntiAffinityPreset: soft
+podAntiAffinityPreset: ""
```

Comment:
Are you sure you need to change this, because as explained in the comment above:
> Ignored if `affinity` is set

If not required, I suggest to keep the original value, so that we limit the BBC-patches to a minimum to simplify future maintenance.